### ISSUE TITLE
Regions will not add

### ISSUE THREAD

**Issue by geomodules1** [geodesicsolutions-community/geocore-community#208]
Unique name (sadasdasd) already in use, not able to add new region

When adding a region it does not work.

**Comment by ghost** [comment]
Same bug's present in V18.  You will need to rewrite the checkUnique function handling this, notably the ADOdb Count statement is insufficient to address the boolean issues being presented there.

**Comment by geomodules1** [comment]
Would you know which file?

**Comment by ghost** [comment]
admin/regions.php

**Comment by geomodules1** [comment]
Have you made the change? If so can you share?

**Comment by geosmarty** [comment]
Yes, but I'm only here to observe bugs and basic comment.  There is a developer associated to this project.

**Comment by geosmarty** [comment]
Sorry I cannot.

**Comment by geosmarty** [comment]
Hi yeah the changes were actually put into the geocoremax commercial project over at geocoremax.com, it's all fixed.  Is not a trivial fix though, requires complete reworking on function and a handful of other calls in that class.  Just really checking in here for bugs.

**Comment by geosmarty** [comment]
lots of ghosts in this project ........

**Comment by geosmarty** [comment]
Deleting comments pointing to other projects and helpful resources is not what opensource or github's about.  This geo-community venture birthed on a development forum freely promoting itself without censure, don't play the hypocrite.

**Comment by geosmarty** [comment]
@geomodules1 ... #204 I have made the changes but I can't share them sorry, the're for a project elsewhere.

**Comment by geosmarty** [comment]
@geomodules1 ... changes have been made, but my comment got deleted for saying where the changes were being made available.  So sorry not available here.

**Comment by geosmarty** [comment]
1.  I'm going to point out to the person deleting comments on this thread, that mentioning other projects or helpful resources is not a breach of opensource @ github or a breach of any code of conduct pertaining to this repository.  This geo-community venture exists because of starting itself on a "development" forum with free promotion.

2.  You responded straight with a level 3 - 4 ban !  Do we need to refresh ourselves on the rules?  

3.  Temporary Ban

Community Impact: A serious violation of community standards, including sustained inappropriate behavior.

Consequence: A temporary ban from any sort of interaction or public communication with the community for a specified period of time. No public or private interaction with the people involved, including unsolicited interaction with those enforcing the Code of Conduct, is allowed during this period. Violating these terms may lead to a permanent ban.

5. Permanent Ban

Community Impact: Demonstrating a pattern of violation of community standards, including sustained inappropriate behavior, harassment of an individual, or aggression toward or disparagement of classes of individuals.

Consequence: A permanent ban from any sort of public interaction within the community.

**Comment by geosmarty** [comment]
I'm going to point out to the person deleting comments on this thread and banning members outright is pathetic.  Mentioning other projects or helpful resources is not a breach of opensource @ github or a breach of any code of conduct pertaining to this repository.  This geo-community venture "exists" from starting itself on a "development" forum with free promotion.   Maybe go check the code of conduct for procedure [https://github.com/geodesicsolutions-community/geocore-community/blob/03b6a39b76821863274e4cfdf1ba8e9faf3854e4/CODE_OF_CONDUCT.md](url)

**Comment by geomodules1** [comment]
support@geomodules.com email me the changes direct please
